### PR TITLE
feat: add 5 min video demo DOCS-469

### PR DESCRIPTION
**Pull request by mrfyda**
this PR leverages the recent video demo from the BB announcement to provide an overview of the product, similar to quality's getting started page: https://docs.codacy.com/getting-started/codacy-quickstart/

### :eyes: Live preview
https://feat-product-demo--docs-pulse.netlify.app/

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Pulse displays the following metrics to provide insights into the current and hi
 -   [Work in progress metrics](metrics/accelerate-wip.md)
 -   [Lead time and cycle time](metrics/lead-cycle-time.md)
 
+Check out our product demo for an overview of Pulse's main features (recorded on January 16, 2023):
+
+<iframe src="https://www.loom.com/embed/8f03ed75a55843149f8abc19ef3fe4be" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
```
Comment (mrfyda) [inline docs/index.md:12]:
```suggestion
<div style="position: relative; padding-bottom: 61.224489795918366%; height: 0;"><iframe src="https://www.loom.com/embed/8f03ed75a55843149f8abc19ef3fe4be" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```

**Comment by mrfyda** [comment]
we can also use the 2 min demo 👀

**Review by claudiacarpinteiro — APPROVED** [review]
@mrfyda Looks great! This video provides the introduction and context about Pulse we were missing. 🚀 
I would keep the 5m video, as it gives more examples of Pulse use cases.

**Review by prcr — APPROVED** [review]

**Review by prcr — COMMENTED** [review]
Really nice idea! :rocket:

**Pull request merged**